docs/source/ops/requirements.rst

```diff
@@ -26,3 +26,8 @@ Daml is tested using the following specific dependencies in default installation
 2. `Eclipse Adoptium <https://adoptium.net>`_ version 11 for Java.
 3. PostgreSQL 10.0
 4. Oracle Database 19.11
+
+Feature/Component System Requirements
+-------------------------------------
+
+1. :doc:`The JavaScript Client Libraries <../app-dev/bindings-ts/index>` is tested on Node 14.18.3. A version greater or equal to 14.18.3 is recommended. 
```

Comment:
IIRC the TS bindings are a browser-side component. Why is Node required?

Reply:
You tell me, I'm just reflecting a requirement from the GSG: https://github.com/digital-asset/daml/pull/13011/files#diff-7efe8ff9244239f5d653233b6f0e0f9f82cbf1dba65b84b780d9268e84b975d3R36

Reply:
I guess we could argue that you don't technically have to use node at all to build a project with the bindings. But you do need the TypeScript compiler, right? Do we want to say anything about the recommended version of that?

Reply:
You do need node in the getting started guide to run create-react-scripts & similar stuff and also to run the typescript compiler. You don’t need node to run the resulting js bundle in your browser.

Reply:
> You tell me, I'm just reflecting a requirement from the GSG: https://github.com/digital-asset/daml/pull/13011/files#diff-7efe8ff9244239f5d653233b6f0e0f9f82cbf1dba65b84b780d9268e84b975d3R36

The Getting Started Guide does use `npm`, so the requirement makes sense there. This doesn't mean one is forced to use it.

Reply:
> You do need node in the getting started guide to run create-react-scripts & similar stuff and also to run the typescript compiler. You don’t need node to run the resulting js bundle in your browser.

Are there requirements with regards to the TypeScript compiler version needed to compile generated code?

Reply:
I added some. Could you confirm you are OK with how it is now?

Reply:
LgTm